LuaMenu/widgets/gui_battle_room_window.lua

```diff
@@ -1291,25 +1291,25 @@ local function InitializeSetupPage(mainWindow, pageConfig, nextPage, selectedOpt
 	}
 	nextButton:Hide()
 
+	advButton = Button:New {

```

Comment:
I don't mind the wizard being there by default, but just want to be able to skip it by clicking on the advanced button immediately from the first page. It feels unintuitive to go to settings to do that, as it interrupts the user flow.